ui/app/AppLayouts/Chat/ChatColumn/ChatInput.qml

```diff
@@ -276,18 +289,50 @@ Rectangle {
         }
     }
 
+    Row {
+        id: messageLengthLimitContainer
+        anchors.right: parent.right
+        anchors.rightMargin: Style.current.padding
+        anchors.verticalCenter: parent.verticalCenter
+        leftPadding: Style.current.halfPadding
+        anchors.verticalCenterOffset: -18
+
+        StyledText {
+            id: messageLengthLimit
+            text: (messageLimit - txtData.length).toString()
+            color: (messageLimit - txtData.length >= 0) ? Style.current.success : Style.current.danger
```

Comment:
What do you think about leaving the color as a secondary text color, and only making it red once the limit has been reached? Ie,
```qml
color: (messageLimit - txtData.length >= 0) ? Style.current.secondaryText : Style.current.danger
```
@simonricoo thoughts?

Reply:
Sounds good. I've made the change and also added a visible attribute that displays the character limit only when it has been surpassed.